src/main/repository/UserRepository.ts

```diff
@@ -9,6 +9,10 @@ export class UserRepository extends Repository<User> {
     return this.findOne({ firstName, lastName });
   }
 
+  findByFirebaseId(idFirebase: string): Promise<User | undefined> {
+    return this.findOne({ firebaseId: idFirebase });
```

Comment:
```suggestion
  findByFirebaseId(firebaseId: string): Promise<User | undefined> {
    return this.findOne({ firebaseId });
```

`{ firebaseId }` is shorthand for `{ firebaseId: firebaseId }`